public/manifest.firefox.json

```diff
@@ -0,0 +1,38 @@
+{
+  "manifest_version": 3,
+
+  "name": "OpenAI Translator",
+  "description": "OpenAI-Translator is a Chrome extension that uses the ChatGPT API for translation.",
+  "version": "1.0",
+
+  "options_ui": {
+    "page": "options.html"
+  },
+
+  "action": {
+    "default_icon": "icon.png",
+    "default_popup": "popup.html"
+  },
+
+  "content_scripts": [
+    {
+      "matches": ["<all_urls>"],
+      "all_frames": true,
+      "js": ["js/content_script.js"]
+    }
+  ],
+
+  "background": {
+    "scripts": ["js/background.js"]
+  },
+
+  "permissions": ["storage"],
+
+  "host_permissions": ["https://*.openai.com/"],
+
+  "browser_specific_settings": {
+    "gecko": {
+      "id": "yetone-openai-translator@example.com"
```

Comment:
`id` needs to be changed.
ref: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings